pkg/cloud/vsphere/actuators/cluster/actuator.go

```diff
@@ -112,6 +117,14 @@ func (a *Actuator) Delete(cluster *clusterv1.Cluster) (opErr error) {
 			"error deleting kubeconfig secret for cluster %q", ctx)
 	}
 
+	// Delete the control plane config map.
+	controlPlaneConfigMapName := actuators.GetNameOfControlPlaneConfigMap(ctx.Cluster.UID)
+	if err := ctx.CoreClient.ConfigMaps(ctx.Cluster.Namespace).Delete(controlPlaneConfigMapName, &metav1.DeleteOptions{}); err != nil {
+		if !apierrors.IsGone(err) && !apierrors.IsNotFound(err) {
+			return errors.Wrapf(err, "error deleting control plane config map %q for %q", controlPlaneConfigMapName, ctx)
```

Comment:
Curious why no requeue error here

Reply:
Huh. Fair. May be worth adding one.

Reply:
I double checked, and it's because the cluster controller doesn't requeue cluster deletes.